Peer.Tests/ConnectorAPI/PeerPullRequestAPITests.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Moq;
+using Peer.Connectors;
+using Peer.Domain;
+using Peer.Domain.Models;
+using Xunit;
+
+namespace Peer.ConnectorApi.Tests
+{
+    public abstract class PeerPullRequestAPITest<T> where T : class
+    {
+        public static void GetPullRequestsTest()
+        {
+            var mockGithubPullRequestSource = new Mock<IPullRequestSource<T>>();
+            var mocklist = new Mock<IEnumerable<PeerPullRequest>>();
+            var mockpr = new PeerPullRequest("title", "CreatedBy");
+            var listpr = new List<PeerPullRequest> { mockpr };
+            mockGithubPullRequestSource.Setup(x => x.FetchPullRequests()).Returns(Task.FromResult<IEnumerable<PeerPullRequest>>(listpr));
+            var _peerPullRequestAPI = new PeerPullRequestAPI<T>(new List<IPullRequestSource<T>> { mockGithubPullRequestSource.Object });
+
+            var res = _peerPullRequestAPI.GetPullRequests();
+            System.Console.WriteLine("asdasdad");
+            Assert.Equal("title", res.Result[0].Title);
+            Assert.Equal("CreatedBy", res.Result[0].Assignee);
+        }
+    }
+
+    public class PeerPullRequestApiTests
+    {
+        [Fact()]
```

Comment:
If you don't supply params you dont need to have ()

Reply:
done